docs/features/opencdc-record.mdx

```diff
@@ -56,9 +80,84 @@ When processing records in Conduit, you can always expect a similar structure to
 ```
 
 :::note
-`.Position`, `.Key`, and `.Payload.Before` are represented as `Base64` encoded in the example above because these will be a byte slice when represented as JSON.
+`.Position`, `.Key`, and `.Payload.Before` are represented as `Base64` encoded
+in the example above because these will be a byte slice when represented as
+JSON.
 :::
 
+## Data types
+
+
+There are no limitations when it comes to data types a source connector can read
+from a source. Certain type limitations might apply depending on how the data is
+moved internally in Conduit, from source connector to Conduit and from Conduit
+to destination connectors.
+
+The key and the payload data can be represented in two ways: raw and structured.
+
+## Raw data
+
+Raw data in Conduit is an array of bytes that can represent the source data in
+its original form (such as a file) or encoded in a certain way (for example, a
+JSON representation of a database row or an Avro encoded structure). It's
+represented by the `opencdc.RawData` Go type. For example, when writing a
+connector or a processor in Go, you can construct raw data like this:
+
+```go
+opencdc.RawData([]byte{1, 3, 5})
+```
+
+## Structured data
+
+Structured data in Conduit is a map in which the keys are field names and values
+are field values. An example of that is the below record's `.Payload.After`
+field: 
+
+
+```json5
```

Comment:
```suggestion
```json
```

Reply:
With `json5` there are no complaints about the comments in JSON:) (in GitHub or in the editor).